### PR TITLE
Feat enable network log asserts

### PR DESCRIPTION
**Pull request by joshuef**
Idea here is basically to provide some helper for grepping the testnet logs for certain things.

### CODE REVIEW

File: .github/workflows/pr_tests.yml [hunk 1]
```diff
@@ -208,23 +208,29 @@ jobs:
...
+      #     killall sn_node
+      #     rm -rf ~/.safe/node
+      #   shell: bash
+
+      # - name: Kill the current network (not needed for next test)
+      #   if: matrix.os == 'windows-latest'
+      #   run: |
+      #     taskkill /IM sn_node.exe /F
+      #     rd /s /q %USERPROFILE%\.safe\node
+      #   shell: cmd
```
Comment (connec) [inline .github/workflows/pr_tests.yml:233]: How come these are commented out?

**Review by connec — COMMENTED** [review]
LGTM, one Q about some commented out stuff in the test workflow (also clippy ain't happy and some tests are failing, can't easily see which/why as the logs are too long).

**Comment by connec** [comment]
In fact, MacOS is failing a new assertion: https://github.com/maidsafe/safe_network/pull/489/checks?check_run_id=3723267210

File: .github/workflows/pr_tests.yml (hunk 1 above)
Comment (joshuef) [reply .github/workflows/pr_tests.yml:233]: Ah, cos the next test is commented out still also. I'll leave that out actually to tidyup the pr

**Comment by joshuef** [comment]
seems like there's a bug and we're still resending more on some AE instances than we'd want 🕵️

**Pull request merged**